src/App.js

```diff
@@ -33,30 +29,25 @@ function App() {
           name: product.name,
           price: product.price,
           count: 1,
+          limit: product.limit,
```

Comment:
So, this work. But instead of adding each property one by one, why not just adding product and just specifying the `count`? 

So something like: 
`setCart([
   ...cart, 
   {
       ...product, 
       count: 1
     }
   ])`
   
   This is cleaner, right?